packages/additionalapi/src/controllers/AdditionalApi.ts

```diff
@@ -1,27 +1,27 @@
 import { Request, Response } from "express"
-import { ADDITIONAL_API_WORKER } from "./AdditionalApiWorker.js"
-import {logger} from "../logging.js";
+import { AdditionalApiContext } from "../main.js"
+import { logger } from "@lionweb/repository-dbadmin"
 
 export interface AdditionalApi {
     getNodeTree(req: Request, res: Response): void
 }
 
-class AdditionalApiImpl implements AdditionalApi {
+export class AdditionalApiImpl implements AdditionalApi {
+    constructor(private context: AdditionalApiContext) {
+    }
     /**
      * Get the tree with root `id`, for one single node
      * @param req
      * @param res
      */
-    async getNodeTree(req: Request, res: Response) {
+    getNodeTree = async (req: Request, res: Response)=> {
```

Comment:
Is not having a space before the `=>` intentional? Unfortunately, `prettier` doesn't seem to have an option for this – at least not specifically for double arrows.

Reply:
No space is not intentional.

Reply:
Using an arrow function is intentional, as it will bind "this" correctly.

Reply:
Websytorm show the following, making it almost impossible to see whether there is a space before the arrow:
<img width="594" alt="image" src="https://github.com/LionWeb-io/lionweb-repository/assets/818164/a942b033-3323-41a5-8d4f-37d76dd0055e">
The " : Promise<void> " is projected by the editor, it is not in the actual source code.

Reply:
Ah yes, and same thing (sometimes) with argument lists. But don't worry: this is just _my_ OCD ;)